gc/base/standard/CopyScanCacheStandard.hpp

```diff
@@ -71,7 +71,7 @@ class MM_CopyScanCacheStandard : public MM_CopyScanCache
 	MMINLINE bool
 	isSplitArray() const
 	{
-		return (OMR_SCAVENGER_CACHE_TYPE_SPLIT_ARRAY == (flags & OMR_SCAVENGER_CACHE_TYPE_SPLIT_ARRAY));
+		return (OMR_COPYSCAN_CACHE_TYPE_SPLIT_ARRAY == (flags & OMR_COPYSCAN_CACHE_TYPE_SPLIT_ARRAY));
```

Comment:
Please use `OMR_ARE_ALL_BITS_SET()`.

Reply:
> Please use `OMR_ARE_ALL_BITS_SET()`.

Functions like this are not used in GC code, this is VM specific review requirement. I am not sure we should start to do it in this change